site/soc/software/config/versions.yaml

```diff
@@ -31,11 +36,40 @@ data:
         reference: 8f290d3b8d2a8d787f107d05b3be0199d33d7026
         subpath: helm-toolkit
         type: git
+      keystone:
+        location: https://git.openstack.org/openstack/openstack-helm
+        reference: 229db2f1552b2bfb0beece4ecd4a1b11eac690c0
```

Comment:
why this reference? is this better than master? Is it a known working freeze? Some comments here would really be appreciated for those of us that are not up to date with the airship meetings :)

Reply:
master is a moving target and is not always validated within airship upstream. This is the airship approach, with an uplifted version, and we override some of the components if we need a different version of some services.

Reply:
tbf this was clarified to me by Matthew as it seems that arsihip pulls its own helm repos instead of using the ones deployed by the osh playbooks, so this makes sense :)